apps/editing-toolkit/editing-toolkit-plugin/full-site-editing-plugin.php

```diff
@@ -145,7 +145,7 @@ function load_timeline_block() {
 function load_common_module() {
 	require_once __DIR__ . '/common/index.php';
 }
-add_action( 'plugins_loaded', __NAMESPACE__ . '\load_common_module' );
+add_action( 'plugins_loaded', __NAMESPACE__ . '\load_common_module', 9 );
```

Comment:
making the common modules load before the other modules has been needed for a while I think.

I can't see any negative impact of this, there's nothing in the common module that depends on anything else and several modules depend on the common module being loaded 👍 

I find the number 9 priority not intuitive here, I think it could use a comment explaining that this loads common modules before other modules.

Reply:
Yes I think it deserves a comment 👍